### PR TITLE
make update-otel

### PR DESCRIPTION
**Pull request by codeboten**
I'll open follow up issues to ensure `Validate` does what is expected in each exporter that wasn't already implementing the method. Didn't want this PR to include all those changes, so instead I added an empty Validate method where needed.

### CODE REVIEW

File: exporter/awskinesisexporter/config.go [hunk 1]
```diff
@@ -45,4 +45,8 @@ type Config struct {
 	MaxRecordSize      int       `mapstructure:"max_record_size"`
 }
 
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/awskinesisexporter/config.go:48]: [exporter/awskinesis] Ensure Validate implementation does the right thing

File: exporter/dynatraceexporter/config/config.go [hunk 2]
```diff
@@ -76,3 +76,7 @@ func (c *Config) ValidateAndConfigureHTTPClientSettings() error {
 
 	return nil
 }
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/dynatraceexporter/config/config.go:80]: [exporter/dynatrace] Ensure Validate implementation does the right thing

File: exporter/googlecloudexporter/config.go [hunk 3]
```diff
@@ -44,6 +44,10 @@ type Config struct {
 	MetricConfig MetricConfig `mapstructure:"metric"`
 }
 
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/googlecloudexporter/config.go:47]: [exporter/googlecloud Ensure Validate implementation does the right thing

File: exporter/honeycombexporter/config.go [hunk 4]
```diff
@@ -39,3 +39,7 @@ type Config struct {
 	// QueueSettings enable queued processing
 	exporterhelper.QueueSettings `mapstructure:"sending_queue"`
 }
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/honeycombexporter/config.go:43]: [exporter/honeycomb] Ensure Validate implementation does the right thing

File: exporter/influxdbexporter/config.go [hunk 5]
```diff
@@ -45,3 +45,7 @@ type Config struct {
 	// - telegraf-prometheus-v2
 	MetricsSchema string `mapstructure:"metrics_schema"`
 }
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/influxdbexporter/config.go:49]: [exporter/influxdb] Ensure Validate implementation does the right thing

File: exporter/lokiexporter/config.go [hunk 6]
```diff
@@ -40,7 +40,7 @@ type Config struct {
 	Format string `mapstructure:"format"`
 }
 
-func (c *Config) validate() error {
+func (c *Config) Validate() error {
```
Comment (codeboten) [inline exporter/lokiexporter/config.go:43]: [exporter/loki] Ensure Validate implementation does the right thing

File: exporter/sapmexporter/config.go [hunk 7]
```diff
@@ -58,7 +58,7 @@ type Config struct {
 	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
 }
 
-func (c *Config) validate() error {
+func (c *Config) Validate() error {
```
Comment (codeboten) [inline exporter/sapmexporter/config.go:61]: [exporter/sapm] Ensure Validate implementation does the right thing

File: exporter/signalfxexporter/config.go [hunk 8]
```diff
@@ -203,3 +203,7 @@ func (cfg *Config) Unmarshal(componentParser *config.Map) (err error) {
 
 	return nil
 }
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/signalfxexporter/config.go:207]: [exporter/signalfx] Ensure Validate implementation does the right thing

File: exporter/splunkhecexporter/config.go [hunk 9]
```diff
@@ -133,3 +133,7 @@ func (cfg *Config) getURL() (out *url.URL, err error) {
 
 	return
 }
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/splunkhecexporter/config.go:137]: [exporter/splunkhec] Ensure Validate implementation does the right thing

File: exporter/sumologicexporter/config.go [hunk 10]
```diff
@@ -133,3 +133,7 @@ const (
 	// DefaultGraphiteTemplate defines default template for Graphite
 	DefaultGraphiteTemplate string = "%{_metric_}"
 )
+
+func (cfg *Config) Validate() error {
```
Comment (codeboten) [inline exporter/sumologicexporter/config.go:137]: [exporter/sumologic] Ensure Validate implementation does the right thing

File: exporter/f5cloudexporter/config_test.go [hunk 11]
```diff
@@ -66,6 +66,7 @@ func TestLoadConfig(t *testing.T) {
 				Headers: map[string]string{
 					"User-Agent": "opentelemetry-collector-contrib {{version}}",
 				},
+				Compression: "gzip",
```
Comment (codeboten) [inline exporter/f5cloudexporter/config_test.go:69]: @gramidt please confirm this change is ok, it is a result of https://github.com/open-telemetry/opentelemetry-collector/commit/5470057bdfacd9821572bbb2c7cd00e5e4fae8e9

**Review by dmitryax — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Review by gramidt — APPROVED** [review]

File: exporter/sumologicexporter/config.go (hunk 10 above)
Comment (pmm-sumo) [reply exporter/sumologicexporter/config.go:137]: OK, I think that works for start and then some part of the "initExporter" will need to be extracted as the validation function. I am going to create a separate PR (I guess after this one is merged?)

**Review by pmm-sumo — APPROVED** [review]

**Pull request merged**